compiler/merge.cc

```diff
@@ -53,6 +53,71 @@ bool MaybeMergeSplitConcat(Graph* graph, Node* node) {
     return true;
 }
 
+bool MaybeMergePadConv(Graph* graph, Node* pad) {
+    if (pad->value() != 0.0 || pad->mode() != "constant") {
+        return false;
+    }
+
+    auto const& pads = pad->pads();
+
+    // range check
```

Comment:
Could you make this comment a complete sentence? For example, "Padding for non-spatial dims cannot be merged."

I forgot to mention, but I'm trying to follow ChainerX's coding guideline (https://github.com/chainer/chainer/blob/master/docs/source/chainerx/contribution.rst#coding-standards), which is based on Google's. 
 Google's says comments should be complete sentences (https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar).

Reply:
OK. I'll follow it!